package.json

```diff
@@ -26,22 +26,24 @@
     "3d"
   ],
   "workspaces": [
-    "modules/*"
+    "modules/core",
```

Comment:
So this will no longer yarn install in my other modules, like babel-plugin-... etc?

Why the change? Do both yarn and lerna use this field?

Reply:
This is shared by lerna and yarn. The `modules/*` directory needs clean up. Subdirectories without `package.json` or `build` scripts break build.